rootfs/chainid.go

```diff
@@ -0,0 +1,50 @@
+package rootfs
+
+import digest "github.com/opencontainers/go-digest"
+
+// NOTE(stevvooe): These are copy-pasted from the opencontainers/image-spec
+// project. Once https://github.com/opencontainers/image-spec/pull/486 is
+// merged, we can use those from the upstream.
+
+// ChainID takes a slice of digests and returns the ChainID corresponding to
```

Comment:
This is the best explanation I could formulate: https://github.com/opencontainers/image-spec/issues/482#issue-194155285. This will land in the specification.